packages/upswyng-server/src/routes/index.svelte

```diff
@@ -14,7 +14,6 @@
   }
 </style>
 
-`
```

Comment:
ummm wut

Reply:
Ya I'm not sure where this came from but that's where that space on the home page is coming from 😅